src/web/src/components/Posts/Post.tsx

```diff
@@ -98,6 +101,15 @@ const useStyles = makeStyles((theme: Theme) =>
         marginLeft: '.1em',
       },
     },
+    postCount: {
+      fontSize: '.8em',
+      marginLeft: '16px',
+      marginRight: '16px',
+      [theme.breakpoints.down(1205)]: {
```

Comment:
The post count on mobile looks a little too close to the title of the post. Probably want to put a bit of margin down there as well